kubernetes/hack/release.ps1

```diff
@@ -0,0 +1,3 @@
+mkdir _out
+$manifest = (Get-Location).Path + "\template\manifest.json"
+hack\packager\AzureGalleryPackager.exe -m $manifest -o _out
```

Comment:
Please download the files from : aka.ms/azurestackmarketplaceitem 
or http://download.microsoft.com/download/4/1/D/41D2B78E-48F0-4FB7-A69F-75A161E595D6/Azure%20Stack%20Marketplace%20Item%20Generator%20and%20Sample.zip